CHANGELOG.md

```diff
@@ -1,16 +1,125 @@
-# Changelog
+## [0.2.0] - 2025-09-13
 
-All notable changes to this project will be documented in this file.
+### Features
 
-## [0.1.0] - First Usable Release
+- Add BeSorted assertion
+- Add BeSameTime assertion for time comparisons
+- Add BeError, BeErrorIs and BeErrorAs assertions for error
+- Add formatted message support for assertions
+- Add NotBeError assertion to verify no error occurs (#38)
```

Comment:
Why does this line is apparently the only one with a reference to an issue number ?